app/controllers/follows_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FollowsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_question, only: [:create, :destroy]
+
+  authorize_resource
```

Comment:
Authorization is falsey: you don't have `@follow` object here, so it just authorizes `Follow` as a class (i.e. simply allows everything). That's a popular CanCan caveat(

And that's why your controller spec passes (although it should not).